controllers/clustergroupupgrade_controller.go

```diff
@@ -164,6 +170,10 @@ func (r *ClusterGroupUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.
 					statusMessage = "The ClusterGroupUpgrade CR has all clusters already compliant with the specified managed policies"
 					nextReconcile = ctrl.Result{RequeueAfter: requeueAfter}
 				} else {
+					if clusterGroupUpgrade.Status.SafeResourceNames == nil {
+						clusterGroupUpgrade.Status.SafeResourceNames = make(map[string]string)
+					}
+					r.Log.Info("[Reconcile]", "name map", clusterGroupUpgrade.Status.SafeResourceNames)
```

Comment:
```suggestion
					r.Log.Info("[Reconcile]", "Status.SafeResourceNames", clusterGroupUpgrade.Status.SafeResourceNames)
```
or something similar. I don't think "name map" is explanatory enough.

Reply:
This was added while I was debugging the issue where reconcile calls work on stale CGU status. I meant to remove it during cleanup. Let me know if you prefer to keep it. BTW, I had added the resource version log later. It's easy to tell if it's happening or not from comparing that log line across multiple reconcile calls.

Reply:
I think we can keep it for now.